src/random_bq_to_anndata.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, cas_feature_index, original_feature_id, feature_name):
         self.original_feature_id = original_feature_id
         self.feature_name = feature_name
 
-# assumes that cas_cell_info.cas_cell_index values are a contiguous list of ints
+# assumes that cas_cell_info.cas_cell_index values are a continuous list of ints
```

Comment:
I can't comment below… but in this line `list(range(min_cas_cell_index, max_cas_cell_index + 1))` just consider much memory this will consume for a bigger dataset?  Might be fine

Reply:
Since I've re-implemented using the in-database randomization/selection than this point is no longer relevant.